### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by RajeshTechForge** [RajeshTechForge/pinac_workspace#10] [General]
Hey @kairveeehh,

We're excited about the progress of our project and believe your expertise in front-end development could greatly enhance our efforts. Would you be interested in contributing with us and making a significant impact on the front-end aspects of our project?

Looking forward to your response.

Best regards,
@RajeshTechForge

**Comment by that-avg-dev** [comment]
yes sure , would like to explore the codebase as well and if there is a deployed site please let me know .

**Comment by RajeshTechForge** [comment]
**That’s awesome!** I made a React component from scratch because I know a bit about React, though I am not good at it. Your expertise in frontend will make the development process much faster.

We haven’t deployed our App yet. We’re planning to deploy it once we have the first full version ready.

**Comment by RajeshTechForge** [comment]
Hi @kairveeehh, While exploring the codebase, if you encounter any doubt or have any other questions, please let me know.

**Comment by that-avg-dev** [comment]
yea as a first year student , I am working on it and getting to know what the project is about . will let you know for doubts

**Reply by RajeshTechForge** [reply]
> yea as a first year student , I am working on it and getting to know what the project is about . will let you know for doubts

That's alright 👍

**Comment by that-avg-dev** [comment]
hello 
@RajeshTechForge  I might not be able to commit to the project due to ongoing programs and self projects . 
really sorry for the inconvenience and would love to collaborate some other time , I am preparing for LFX so being a bit busy rn.

**Reply by RajeshTechForge** [reply]
That's completely fine.
